packages/js/components/src/table/empty.tsx

```diff
@@ -1,39 +1,32 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import { createElement } from '@wordpress/element';
+import React from 'react';
+
+type EmptyTableProps = {
+	children: React.ReactNode;
+
+	// An integer with the number of rows the box should occupy.
+	numberOfRows: number;
```

Comment:
Looks like there is a default value for `numberOfRows`. Shall we change this to be optional?

```suggestion
	numberOfRows?: number;
```